cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -375,6 +376,35 @@ std::array<std::array<std::vector<char>, NUM_SYMBOL_GROUPS>, TT_NUM_STATES> cons
 
 // The DFA's starting state
 constexpr auto start_state = static_cast<StateT>(TT_OOS);
+
+template <typename SymbolT>
+auto get_sgid_lut(SymbolT delim)
+{
+  symbol_groups[static_cast<std::uint8_t>(dfa_symbol_group_id::DELIMITER_CHAR)] = {delim};
+  return symbol_groups;
+}
```

Comment:
suggestion: To keep the definition in a single place, I was thinking of something along the lines of:
```suggestion
template <typename SymbolT>
auto get_sgid_lut(SymbolT delim)
{
  return std::array<std::vector<SymbolT>, NUM_SYMBOL_GROUPS - 1>{{{'{'}, {'['}, {'}'}, {']'}, {'"'}, {'\\'}, {delim}}};
}
```

Reply:
Thanks, this is a much cleaner way to implement it!